.circleci/config.yml

```
@@ -1,6 +1,63 @@
 version: 2.1
 orbs:
   node: circleci/node@4.2.1
+commands:
+  set-environment:
+    parameters:
+      env:
+        default: "DEV"
+        type: "string"
+    steps:
+      - run: |
+        CF_SPACE=CF_SPACE_<<parameters.env>>
+        CF_USERNAME=CF_USERNAME_<<parameters.env>>
+        CF_PASSWORD=CF_PASSWORD_<<parameters.env>>
+        CGHOSTNAME_FRONTEND=CGHOSTNAME_FRONTEND_<<parameters.env>>
+        CGHOSTNAME_BACKEND=CGHOSTNAME_BACKEND_<<parameters.env>>
+  deploy-frontend:
+    parameters:
+      deploy-strategy:
+        default: "rolling"
+        type: string
+      deploy-env:
+        default: "staging"
+        type: string
+      backend-hostname:
+        default: "https://tdp-backend.app.cloud.gov/"
```

```suggestion
        default: "tdp-backend"
```
looks like the way this gets used it should be just this portion of the hostname, perhaps CG_APPNAME is a better name for the variable.